OrangeHRMTests/Tests/Tests.cs

```diff
@@ -0,0 +1,590 @@
+﻿using NUnit.Allure.Core;
+using NUnit.Framework;
+using NUnit.Framework.Legacy;
+using OpenQA.Selenium;
+using OrangeHRMTests.Common.WebElements;
+using OrangeHRMTests.Data;
+using OrangeHRMTests.Data.Constants;
+using OrangeHRMTests.PageObjects;
+using OrangeHRMTests.PageObjects.Modules;
+
+namespace OrangeHRMTests.Tests
+{
+    [TestFixture]
+    [AllureNUnit]
+    public class DemoQATests : BaseTest
+    {
+        [Test]
+        public void A_AboutModalTest()
```

Comment:
Test name should reflect the main idea of test  rather then be named as A B C etc
I would better name it like VerifyAboutPopupPropertiesValues

Please think about other names for all other tests

Reply:
Fixed